sklearn/calibration.py

```diff
@@ -1298,6 +1298,9 @@ def from_predictions(
         prob_true, prob_pred = calibration_curve(
             y_true, y_prob, n_bins=n_bins, strategy=strategy
         )
+        name = name if name is not None else "Calibrated classifier"
```

Comment:
The classifier does not need to be calibrated, to have a calibrated curve?

```suggestion
        name = name if name is not None else "Classifier"
```

L1254 docstring:

```
name : str, default None
    Name for labeling curve. If `None`, name will be set to "Classifier".
```